### PR TITLE
HTML/CSS & JavaScript capstone project

### PR DESCRIPTION
**Pull request by BahirHakimy**
# First module capstone completed as described in this [Exercise](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)
### In this project I have done the following:
- Implemented all the requirements based on the given design.
- Used semantic HTML tags.
- Applied best practices for HTML.
- Used Flexbox to place elements on the page.
- Used images and backgrounds to enhance the look of a website.
- Deployed the project on GitHub pages.
- Applied JavaScript best practices.
- Used JavaScript events.

### Also here is the link for the loom intro [video](https://www.loom.com/share/a2a41a9ada0b41f68f1c6f6e6c97378b)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,448 @@
...
+        >
+          <div
+            class="flex justify-start md:justify-center md:flex-col gap-2 items-center p-4"
+          >
+            <img
+              class="w-16 h-auto"
+              src="./assets/icons/lecture.png"
+              alt="lecture-icon"
+            />
+            <p class="text-xl font-bold text-primary w-max">Lecture</p>
```
Comment (ShadyShawkat) [inline index.html]: - [ ] Kindly make sure that the text doesn't overlay one another to match the given design.
- [ ] Please remove the white space that appears on the right side of the index page in mobile view.
image

File: README.md [hunk 2]
```diff
@@ -1,2 +1,153 @@
...
+
+If you like this project leave a start for it.
+
+<p align="right"><a href="#readme-top">👆</a></p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for helping me in my journey to become a Fullstack developer 🌹
```
Comment (ShadyShawkat) [inline README.md]: - [ ] Kindly give appropriate credit to the creator of the original design. I've noticed that you mentioned this in the "Description" section of the readme, but writing it here in the "Acknowledgments" section looks more professional as this section was created for this sole purpose.

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @BahirHakimy, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 
- Good overall design ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @BahirHakimy, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: index.html (hunk 1 above)
Comment (BahirHakimy) [reply index.html]: Thanks @ShadyShawkat for your kind feedback, I didn't noticed that, I will make the required changes 👨‍💻

File: README.md (hunk 2 above)
Comment (BahirHakimy) [reply README.md]: Thanks for reminding I will update it, Thank you 😃

**Comment by BahirHakimy** [comment]
> Hi @BahirHakimy,
> 
> In my opinion, your project is ready for the presentation! There is nothing else to say other than... it's time to merge it :shipit:
> 
> Good luck! 🎉
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Thanks for approving my project 💯

**Pull request merged**